issuetracker/pivotaltracker/pivotaltracker.go

```diff
@@ -20,7 +20,7 @@ func (it *IssueTracker) TaskModel() issuetracker.Task {
 
 // TaskURLFrom taskID returns the url for the target pivotaltracker task ID to fetch
 func (it *IssueTracker) TaskURLFrom(taskID string) string {
-	if len(taskID) > 0 && taskID[0] == '#' {
+	if len(taskID) > 0 && strings.HasPrefix(taskID, string('#')) {
```

Comment:
no need for the `len(taskID) > 0` check anymore. `strings.HasPrefix` [deals with that](https://play.golang.org/p/CxSXIt0YoK7).

Reply:
Sure, this makes sense